test/e2e/forced_batches_test.go

```diff
@@ -23,64 +23,112 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const (
+	// dockersArePreLaunched is a flag that indicates if dockers are pre-launched, used for local development
+	// avoiding launch time and reset database time at end (so you can check the database after the test)
+	dockersArePreLaunched = false
+	gerFinalityBlocks     = uint64(9223372036854775807) // The biggeset uint64
```

Comment:
This will cause the GER to never be considered

Reply:
The opposite. This value is send to the state function `GetLatestGer` that use this value to check the blocknumber below that. It also could affect to sequencer in `finalizer.go`. 
Maybe the bug is in the state funcion instead of this constant

Reply:
You are right. Consider using math.MaxUint64 if it works for you.